values.yaml

```diff
@@ -198,6 +198,10 @@ extraVolumeMounts: []
 extraVolumes: []
 
 resources: {}
+  # If you have more than 1 replica, the minimum recommended resources configurate are as follows:
+  # - cpu: 2048m
+  # - memory: 4098Mi
```

Comment:
it's unclear whether these are requests or limits - also if this is what we suggest, can we just modify the existing 500m 500mi comments below?

Reply:
These should be requests. I updated the following requests and doubled the limit on top of that